dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
Yes, the fixing my be enabled by default.

01.09.2016 13:34 "jacob-carlborg" <notifications@github.com> napisał(a):

> In dstep/Configuration.d
> <https://github.com/jacob-carlborg/dstep/pull/95#discussion_r77157149>:
>
> > @@ -77,6 +77,10 @@ struct Configuration
> >      /// do not translate following symbols
> >      @("skip", "Skip translation of <symbol>.")
> >      string[] skipSymbols;
> > +
> > +    /// enable printing of warnings
> > +    @("enable-warnings", "Enable printing of warnings [default].")
> > +    bool enableWarnings = true;
>
> My point is, that there are two orthogonal concepts: issuing a message and
> trying to fix the problem.
>
> That's true. If we start with making issuing a message configurable?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jacob-carlborg/dstep/pull/95/files/f7cc5b685291575bc8bae2c72ce578b2f85431b9#r77157149>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHfoNJ-WVSyNG3OHPRxMi2Wszfj_51OGks5qlrg_gaJpZM4JyGgj>
> .
>